src/olympia/versions/models.py

```diff
@@ -693,6 +693,18 @@ def can_be_disabled_and_deleted(self):
         return True
 
 
+class VersionReviewerFlags(ModelBase):
+    version = models.OneToOneField(
+        Version, primary_key=True, on_delete=models.CASCADE
+    )
+    needs_human_review_by_mad = models.BooleanField(default=False)
+    pending_rejection = models.DateTimeField(default=None, null=True)
+
```

Comment:
We'll lose any existing flags in `VersionScannerFlags` - should we either be setting `Meta.db_table` to point to the old table or have a python migration that copies them over?

Reply:
@willdurand said it's ok to lose the data